File: src/main/java/org/example/TC/김서방.java
package org.example.TC;
/*https://school.programmers.co.kr/learn/courses/30/lessons/12919*/
public class 김서방 {
    public String solution(String[] seoul) {
        String answer = "";
        for(int i=0; i<seoul.length; i++) {
            if(seoul[i].equals("Kim")) answer = "김서방은 " + i + "에 있다";
        }
        return answer;
    }
}

File: src/main/java/org/example/TCK/김서방.kt
package org.example.TCK

import java.util.Arrays

/*https://school.programmers.co.kr/learn/courses/30/lessons/12919*/
class 김서방 {
    fun solution(seoul: Array<String>):String {
        var answer = ""
        var k :Int = 0
        for(i in seoul) {
            if(i.equals("Kim")) {
                 ++k
                answer = "김서방은 "+k+"에 있다"
            }
        }
        return answer
    }
}